thirdparty/tinymce/tiny_mce_gzip.php

```diff
@@ -19,7 +19,7 @@
 	// Default settings
 	$tinyMCECompressor = new TinyMCE_Compressor(array(
 		// CUSTOM SilverStripe
-		'cache_dir' => TEMP_FOLDER
+		'cache_dir' => TEMP_FOLDER.'/tinymce-cache'

```

Comment:
This directory needs to be created first, otherwise the cache is never written 😞